src/fileInput/gridDynRunner.h

```diff
@@ -58,18 +58,18 @@ class GriddynRunner
    * Initialize a simulation run from command line arguments.
    @param[in] argc the number of console arguments
    @param[in] argv the actual console arguments
-   @param[in] ignoreUnrecognized set to true to indicate that the unrecognized arguments should be ignored
+   @param[in] allowUnrecognized set to true to indicate that the unrecognized arguments should be allowed
```

Comment:
had to stare at this for a while to make sure the logic was correct but I like the rename